cuda_bindings/tests/test_cuda.py

```diff
@@ -948,3 +948,44 @@ def test_conditional():
 def test_CUmemDecompressParams_st():
     desc = cuda.CUmemDecompressParams_st()
     assert int(desc.dstActBytes) == 0
+
+
+def test_all_CUresult_codes(max_code=1000):
```

Comment:
btw I think this should work: the number of `IntEnum` can be queried via `len()`
```suggestion
def test_all_CUresult_codes(max_code=len(cuda.CUresult)):
```

Reply:
I want to intentionally run past the end of the enums. Stress test; simpler, too.

Reply:
Ah, ok, so I confused myself (and you, sorry!), sounds like what we need is `range(int(max(cuda.CUresult))+1)` to go beyond the max value 999?

Reply:
Done in commit 781446943cb22130be26dea22d76aef06f0b7ac8.